bundles/org.openhab.persistence.jpa/src/main/java/org/openhab/persistence/jpa/internal/JpaPersistenceService.java

```diff
@@ -164,7 +165,12 @@ public void store(Item item, @Nullable String alias) {
             em.getTransaction().commit();
             logger.debug("Persisting item...done");
         } catch (Exception e) {
-            logger.error("Error while persisting item! Rolling back!", e);
+            if (e.getCause() instanceof EntityExistsException) {
+                // there's a UNIQUE constraint in the database, and we tried to write
+                // a duplicate timestamp. Just ignore
```

Comment:
Perhaps this should still be logged at log level **debug**? Since a value is actually discarded, this is still some kind of error situation, and it might be good to have the opportunity to backtrack this.